### PR TITLE
Feat/pt br translation tabindex

### PR DESCRIPTION
**Pull request by alexalannunes**
Changes proposed in this pull request:

Add pt-BR translation for tabindex on /lighthouse-accessibility - tabindex

When you're ready to submit your PR, don't forget to add the `$-presubmit` label.

### CODE REVIEW

**Comment by jecfish** [comment]
Hi @alvarocamillont, could you help to review this translation? The original english version is here: https://web.dev/tabindex/.

File: src/site/content/pt/lighthouse-accessibility/tabindex/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Se você tiver um `tabindex` maior que `0`,
+e estiver usando um link ou elemento de formulário,
+remova o `tabindex`.
+Elementos HTML como `<button>` ou `<input>`
+tem acessibilidade de teclado integrada nativamente.
+
+Se você estiver usando componentes interativos personalizados,
+defina o `tabindex` para `0`.
+For example:
```
Comment (alvarocamillont) [inline src/site/content/pt/lighthouse-accessibility/tabindex/index.md:40]: Change to "Por exemplo"

File: src/site/content/pt/lighthouse-accessibility/tabindex/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+```html
+<div tabindex="0">Pressione a tecla TAB para focar aqui</div>
+```
+
+Se você precisar que um elemento chegue mais cedo ou mais tarde na ordem de tabulação,
```
Comment (alvarocamillont) [inline src/site/content/pt/lighthouse-accessibility/tabindex/index.md:46]: Change "chegue mais cedo ou mais tarde" to "apareça antes ou depois"

**Review by alvarocamillont — COMMENTED** [review]
Some details that need to be address :)

**Pull request closed without merging**